### PR TITLE
vendor: bump compute deps 140823

### PR DESCRIPTION
**Pull request by Tal-or**
- remove RTE deps
- bump NTO to tip of master's branch - correct due to commit date

### CODE REVIEW

**Comment by Tal-or** [comment]
/hold
depends on https://github.com/openshift/cluster-node-tuning-operator/pull/721

**Comment by yanirq** [comment]
@ffromani  please review for approval

File: go.mod [hunk 1]
```diff
@@ -189,10 +189,10 @@ replace (
 // Test deps
 replace (
 	github.com/k8snetworkplumbingwg/sriov-network-operator => github.com/openshift/sriov-network-operator v0.0.0-20230704134056-a62bc94a10f3 // release-4.14
-	github.com/k8stopologyawareschedwg/resource-topology-exporter => github.com/k8stopologyawareschedwg/resource-topology-exporter v0.8.0
+	github.com/k8stopologyawareschedwg/resource-topology-exporter => github.com/k8stopologyawareschedwg/resource-topology-exporter v0.12.0
```
Comment (ffromani) [inline go.mod:192]: I think we (probably I) just forgot to remove this and it is no longer used

**Review by ffromani — COMMENTED** [review]
seems fine, but we can just remove the RTE dep

**Comment by Tal-or** [comment]
/hold
depends on https://github.com/openshift/cluster-node-tuning-operator/pull/727

**Comment by Tal-or** [comment]
/test e2e-gcp-ovn
/hold cancel
https://github.com/openshift/cluster-node-tuning-operator/pull/727 merged

File: go.mod [hunk 2]
```diff
@@ -189,10 +189,9 @@ replace (
 // Test deps
 replace (
 	github.com/k8snetworkplumbingwg/sriov-network-operator => github.com/openshift/sriov-network-operator v0.0.0-20230704134056-a62bc94a10f3 // release-4.14
-	github.com/k8stopologyawareschedwg/resource-topology-exporter => github.com/k8stopologyawareschedwg/resource-topology-exporter v0.8.0
 	github.com/metallb/metallb-operator => github.com/openshift/metallb-operator v0.0.0-20230123113337-fa1059e25fe1 // release-4.13
 	github.com/openshift-psap/special-resource-operator => github.com/openshift/special-resource-operator v0.0.0-20211202035230-4c86f99c426b // release-4.10
 	github.com/openshift/cluster-nfd-operator => github.com/openshift/cluster-nfd-operator v0.0.0-20210727033955-e8e9697b5ffc // release-4.9
-	github.com/openshift/cluster-node-tuning-operator => github.com/openshift/cluster-node-tuning-operator v0.0.0-20230704170229-287fdce04769 // release-4.14
+	github.com/openshift/cluster-node-tuning-operator => github.com/openshift/cluster-node-tuning-operator v0.0.0-20230726094828-96b067113eca // release-4.15
```
Comment (SchSeba) [inline go.mod:195]: I think it's still 4.14 no?
we didn't branch out yet so please do change this to 4.15

File: go.mod [hunk 3]
```diff
@@ -26,21 +26,21 @@ require (
...
+	k8s.io/api v0.27.4
+	k8s.io/apiextensions-apiserver v0.27.4
+	k8s.io/apimachinery v0.27.4
 	k8s.io/client-go v12.0.0+incompatible
 	k8s.io/klog v1.0.0
-	k8s.io/klog/v2 v2.90.0
-	k8s.io/kubelet v0.26.2
+	k8s.io/klog/v2 v2.100.1
+	k8s.io/kubelet v0.27.4
 	k8s.io/kubernetes v1.26.1
```
Comment (liornoy) [inline go.mod:42]: @Tal-or can you please also bump `k8s.io/kubernetes` to 1.27.4?
This should fix all of the "undefined" errors.

**Comment by Tal-or** [comment]
Closing. superseded by #1634

**Pull request closed without merging**